std_msgs/Header header
int16 uav_id
geometry_msgs/Pose pose
geometry_msgs/Twist velocity
float32 heading
int16 is_sim
